atlasaction/comments/schema-lint.tmpl

```diff
@@ -0,0 +1,38 @@
+<code>atlas schema lint</code> on <strong>{{ join .URL ", " }}</strong>
+{{- if and .Steps (gt (len .Steps) 0) }}
+<table>
+  <thead>
+    <tr>
+      <th>Status</th>
+      <th>Rule</th>
+      <th>Result</th>
+    </tr>
+  </thead>
+  <tbody>{{ range $step := .Steps }}
+    <tr>
+      <td>
+        <div align="center">
+          {{- template "lint-check" "error.svg" -}}
+        </div>
+      </td>
+      <td><b>{{ $step.Text | firstUpper }}</b></td>
```

Comment:
```suggestion
      <td>{{ $step.Text | firstUpper }}</td>
```
<img width="825" alt="image" src="https://github.com/user-attachments/assets/061d046f-46a9-47a1-9375-3ea469fc2728" />

This column is not bold in `migrate lint`. Let's keep the two messages the same